packages/cli/src/lib/codegen.ts

```diff
@@ -76,23 +84,19 @@ export function spawnCodegenProcess({
 
     // Filter these logs even on verbose mode because it floods the terminal:
     if (/`punycode`/.test(message)) return;
-    if (/\.body\[\d\]/) return;
+    if (/\.body\[\d\]/.test(message)) return;
+    if (/console\.time(End)?\(\)/.test(message)) return;
```

Comment:
What was the `console.time` added for?

Reply:
When you run with `--verbose` you start seeing a loooot of  warnings from Codegen that include `.body` or `console.time()`. I guess Codegen dependencies are also looking for a `--verbose` flag or similar, not sure.
Just muting these so that the output in the terminal is useful.

Reply:
Maybe worth a comment for that?

Reply:
There's already one above this diff: `// Filter these logs even on verbose mode because it floods the terminal:`